src/02-common-principles.md

```diff
@@ -644,6 +540,150 @@ for more information.
 }
 ```
 
+## The Inheritance Principle
+
+1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+
+1.  For a given data file, any metadata file is applicable to that data file if:
+    1.  It is stored at the same directory level or higher;
+    1.  The metadata and the data filenames possess the same suffix;
+    1.  The metadata filename does not include any entity absent from the data filename.
+
+1.  A metadata file MUST NOT have a filename that would be otherwise applicable
+    to some data file based on rules 2.2 and 2.3 but is made inapplicable based on its
+    location in the directory structure as per rule 2.1.
```

Comment:
I like these back-references to 2.2, 2.3 etc. -> however in the rendered version, the nested enumerated list has letters, not numbers (see screenshot below)

![image](https://user-images.githubusercontent.com/9084751/146912892-e6911f75-95ff-4384-9891-93260f33fd2d.png)

So I would suggest to refer to 2a, 2b, 2c here.

Reply:
Closed by 8e808a12.